backend/charm/config.yaml

```diff
@@ -8,3 +8,6 @@ options:
     default: test-observer-api
     description: Public hostname for the service.
     type: string
+  sentry_dsn:
+    description: value for SENTRY_DSN that is used to send errors to sentry
+    type: string
```

Comment:
small note, we could have a default value of "" so that it's easier to deploy locally through `juju deploy test-observer-api`